src/client/pages/OfferNew/ComparisonTable/ComparisonTable.tsx

```diff
@@ -13,13 +15,75 @@ import {
   bundleHasPeril,
   getUniquePerilsForQuoteBundles,
 } from 'api/quoteBundleSelectors'
+import { ChevronDown } from 'components/icons/ChevronDown'
+import { ChevronUp } from 'components/icons/ChevronUp'
+import { BREAKPOINTS } from 'utils/mediaQueries'
 
 type ComparisonTableProps = {
   bundles: QuoteBundle[]
 }
 
+const RowTitle = styled.div`
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+
+  svg {
+    margin-left: 0.625rem;
+  }
+`
+
+const RowDescription = styled.p`
+  font-size: 0.875rem;
+  color: ${colorsV3.gray700};
+`
+
+const AccordionRow = styled(TableRow)`
+  cursor: pointer;
+`
+
+const TitleTableCell = styled(TableCell)`
+  max-width: 16rem;
+`
+
+const PerilTableCell = styled(TableCell)`
+  vertical-align: top;
+`
+
+type AccordionProps = {
+  bundles: QuoteBundle[]
+  row: PerilV2
+  isActive: boolean
+  setIsActive: (isActive: boolean) => void
+}
+
+const Accordion = ({ bundles, row, isActive, setIsActive }: AccordionProps) => {
+  return (
+    <AccordionRow key={row.title} onClick={() => setIsActive(!isActive)}>
```

Comment:
If this element is clickable it should be a button or have a role of a button and a focus state for accessibility reasons :)

But feel free to iterate on that in a future PR!